zorch/testing/grind_and_sample_test.py

```
@@ -0,0 +1,80 @@
+    return [fnp.asarray(x).tobytes() for x in frx.tree_util.tree_leaves(t.state)]
+
+
+class FusedPairTest(parameterized.TestCase):
+    def _assert_same(self, unmerged, merged, what):
+        t_a, r_a = unmerged
+        t_b, r_b = merged
+        self.assertEqual(
+            fnp.asarray(r_a).tobytes(), fnp.asarray(r_b).tobytes(),
+            f"challenge differs for {what}",
+        )
+        for i, (x, y) in enumerate(zip(_state(t_a), _state(t_b))):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)